guides/common/modules/con_using-structured-apt-content.adoc

```diff
@@ -0,0 +1,32 @@
+[id="using-structured-apt-content"]
+= Using structured APT content
+
+You can enable structured APT content on your {Project} to use the same Deb repository structure as the remote repositories they are synchronized from.
+
+Features::
+With structured APT content, Deb repositories on {Project} use the same structure as the upstream Deb repository.
+This allows features that attach to the repository structure to work the same way as they would with the upstream repository, for example:
+
+* `apt list --upgradable` will no longer list `unknown,unknown` as the release and distribution for packages.
+Instead, `apt` will list the proper release and distribution from the upstream repository.
+* You can use client-side APT pinning in the same way as when using the upstream repository directly.
+
+Comprehensibility::
+With structured APT, repositories in {Project} simply use the same structure as the upstream repository.
+They do not use the special `default/all` simple publication concept which does not exist outside of {Project}.
```

Comment:
The structuring is good but I'd use the same formatting for these bits as for the Limitations below.

```suggestion
.Comprehensibility
With structured APT, repositories in {Project} use the same structure as the upstream repository.
They do not use the special `default/all` publication concept which does not exist outside of {Project}.
```
Drop "simply". I'm not sure what the last sentence means but it looks overburdened with adjectives.